code/modules/mob/living/carbon/racial_spells.dm

```diff
@@ -0,0 +1,35 @@
+//For spells designed to be inherent abilities given to mobs via their species datum
+
+/spell/racial
```

Comment:
I'd rather people were made to do that small amount of manual work over someone seeing this subtype and deciding all racial spells should go under it before promptly copying the entirety of `spell/targeted` into a new type.